source/process/docs-reviewers-guide.rst

```diff
@@ -0,0 +1,116 @@
+   - Verify RST renders properly in GitHub preview
+
+- **Needs Technical Approval** - Technical change requiring developer review, e.g. change in system requirements.
+
+- **Needs Testing Approval** - Change requires manual testing, e.g. significant install process change. 
+
+- **Needs Sphinx Approval** - Significant structural changes to CSS or Sphinx features, requires HTML generation and verification. 
+
+- **Needs Structural Approval** - Approval required on changes to TOC or organization.
+
+- **Needs Verification & Acknowledgement** - docs.mattermost.com needs to be verified and acknowledgement added to PR.
+
+- **Post-Merge Editing Required** - PR can be merged and someone will follow-up with edits for minor corrections.
+
+- **Awaiting Submitter Action** - Merge should not happen until feedback is addressed. Note: This label should be applied for any work-in-progress PR, a separate label is not needed. 
+
+Levels 
+-------------
+
+The anticipated consistency of reviews by Reviewers is noted using the following levels: 
+
+- **Level 1** - Self-identified expertise in a review criteria, and willing to help.
+- **Level 2** - 80%+ track record of good reviews, or subjective promotion to this level by Level 3 or above. 
+- **Level 3** - 90%+ track record of correct reviews, or subjective promotion to this level by Level 4 or above. 
+- **Level 4** - 95%+ track record of correct reviews or maintainer of doc repo.
+
+Extended periods of consistent performance are recognized by level increases. Repeated inconsistency reduces an reviwers level. 
+
+Roles
+-------------
+
+The following lists roles involved in the PR review process: 
+
+- **Approved Contributor** - Anyone who's completed the CLA. This is the minimum role to submit a PR for review. 
+- **Project Member** - Approved Contributor granted ability to apply labels. Often a staff member from Mattermost, Inc. 
+- **Project Member Reviewer on call** - Member assigned to review PRs in certain approval categories. 
+- **Project Member Verifier on call** - Member assigned to verify and acknowledge merged doc PRs. 
+- **Project Member Editor on call** - Member assigned to edit PRs with "Post-merge edit required".
+
+Approved Contributors who are not Members can participate in the PR review process, and through consistent performance may be promoted to Members: 
+
+- **Contributor Reviewer** - Approved Contributor who volunteers to review PRs prior to Member review. 
+- **Contributor Verifier** - Approved Contributor who volunteers to verify PRs after merge.
+- **Contributor Editor** - Approved Contributor who submits PRs editing merged changes with "Post-merge edit required" label.
+
+Helping identify any items missed by Members is highly welcome and encouraged. 
+
+PR Review Process for Reviewers 
+---------------------------------------
+
+The following process is used determine how reviewers on-call leave feedback on doc PRs submitted: 
```

Comment:
why is it "on-call" here and "on call" above? If you're using it as an adjective, I think it would be "on-call reviewers"?